plugins/sudoers/toke.c

```diff
@@ -5,7 +5,7 @@
 
 #include <config.h>
 
-#line 3 "toke.c"
```

Comment:
Please drop changes to this file, your version of flex and mine appear to disagree on line numbers.